packages/react-core/src/layouts/Alignment/Alignment.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly-next/layouts/Alignment/styles.css';
+import PropTypes from 'prop-types';
+import { componentShape } from '../../internal/componentShape';
+
+const propTypes = {
+  children: PropTypes.any,
+  className: PropTypes.string,
+  component: componentShape,
+  isRight: PropTypes.bool,
```

Comment:
it might make sense to combine these into a single prop since it is not possible to have left and center together.  As for the name I'm not sure haha.  could go with `align`, `alignment`, or `direction`.  align is less typing so i think that might be my vote.  Also `<Alignment alignment=` looks a little odd.

Reply:
@dmiller9911 would this be something like:

<Alignment alignment="left/right/center"

```
const propTypes = {
 ...
  align: PropTypes.string
};

const defaultProps = {
  ...
  align: ''
};

<Component
    className={css(
      styles.alignment,
      className,
      align === 'center' && styles.modifiers.center,
      align === 'right' && styles.modifiers.right,
      align === 'left' && styles.modifiers.left
    )}
    {...props}
  >
    {children}
</Component>
```

Reply:
That is one way for it another would be to do
```
import { .css, getModifier }  from '@patternfly/react-styles';

<Component
    className={css(
      styles.alignment,
      className,
      getModifier(styles, align),
    )}
    {...props}
  >
    {children}
</Component>
```
Both achieve the same goal, but this is a shortcut.  it will also allow us to add other alignments without changing too much code. (although there will probably never be another)